client/landing/stepper/declarative-flow/new-hosted-site-flow.ts

```diff
@@ -55,6 +56,7 @@ const hosting: Flow = {
 		];
 	},
 	useStepNavigation( _currentStepSlug, navigate ) {
+		const siteCount = useSelector( getSites ).length;
```

Comment:
I feel the site count from the user object is going to be more reliable because the full site list is large and takes a long time to load. Feels more likely we'll have the user object loaded.

```suggestion
		const siteCount = useSelector( state => getCurrentUser( state )?.site_count );
```

Reply:
Yeah, I was using that first, but I was afraid of the number not updating if the user doesn't refresh the page. I'll give it another shot and check whether it works as we think it does 🧑‍🔬

Reply:
Oh that's an interesting point. Worth checking.